packages/worker/src/workflow/bundler.ts

```diff
@@ -269,14 +269,15 @@ exports.importInterceptors = function importInterceptors() {
 
             if (this.foundProblematicModules.size) {
               const err = new Error(
-                `Your Workflow code (or a library used by your Workflow code) is importing the following built-in Node modules:\n` +
+                `Your Workflow code (or a library used by your Workflow code) is importing the following black-listed modules:\n` +
                   Array.from(this.foundProblematicModules)
                     .map((module) => `  - '${module}'\n`)
                     .join('') +
                   `Workflow code doesn't have access to built-in Node modules (in order to help enforce determinism). If you are certain ` +
```

Comment:
I think this whole paragraph should be rewritten as a list of things to check instead...

For example:

```suggestion
                  `These modules can't be used in workflow context as they might break determinism.` +
                  `HINT: Consider the following options:\n` +
                  ` • Make sure that activity code is not imported from workflow code. Use \`import type\` to import activity function signature.\n` +
                  ` • Move code that has non-deterministic behaviour to activities.\n` +
                  ` • If you know for sure that a disallowed module will not be used at runtime, add its name to 'WorkerOptions.bundlerOptions.ignoreModules' in order to dismiss this warning.\n` +

```

Reply:
Also, what would really help here is to give the trace back on problematic imports... We definitely need to get there.

Reply:
Yeah, I've been meaning to increase the webpack verbosity on error, never got to that.